options.js

```diff
@@ -112,4 +140,146 @@ function send_message_to_tabs( tabs ) {
 			.catch()
 		// }
 	}
+	if( is_options_were_changed == true ) {
+		is_options_were_changed = false;
+		browser.tabs.sendMessage(
+			tabs[0].id,
+			{ recreate: "true" }
+		).catch();
+	}
+}
+
+// Auto update clock's css after saving changes.
+function message_options_were_changed() {
+	is_options_were_changed = true;
+	get_active_tab()
+	.then( send_message_to_tabs )
+	.catch( function (error) {
+		console.error( `overlay_clock message_option_were_changed tab_query error: ${error}` );
+		});
+}
+
+function switch_options_page( to_hide, to_show ) {
+	// console.log( "switch_options_page " + to_hide + " " + to_show + " worked" );
+	document.getElementById( to_hide ).style.display = "none";
+	document.getElementById( to_show ).style.display = "block";
+}
+
+
+// For each saved domain create an div with it and add to page_domains_list.
+function recreate_domains_list_in_html( domains ) {
+	domains.forEach( function( value ) {
+		domains_container.appendChild( create_domain_element_div( value ) );
+	})
+}
+
+// helper function
+// Returns new element.
+function create_domain_element_div( value ) {
+	let domains_container = document.getElementById( "domains_container" );
+	let new_element = document.createElement( "div" );
+	// Or classList.add( "domain_element" ); if there some else classes
+	new_element.className = "domain_element";
+
+	let domain_input = document.createElement( "input" );
+	domain_input.value = value;
+	domain_input.type = "text";
+	domain_input.placeholder = "www.domain.name";
+	domain_input.className = "domains";
+	// both work
+	// domain_input.setAttribute( "readonly", true);
+	domain_input.readOnly = true;
+
+	let domain_button = document.createElement( "input" );
+	domain_button.type = "button";
+	domain_button.title = "Delete this domain";
+	domain_button.value = "❌";
```

Comment:
I'm not sure where the "+" is coming from (to add a domain) but please replace it with "➕".

Reply:
> "➕"

Are you sure? While browsers have inner emoji fonts (or whatever has this symbol). My casual fonts don't have the symbol of yours, so i have it in text editor only as 2795 in square.

Reply:
Yes, I'm sure. I see it correctly on FF and Chrome (Mac) as well as FF on Ubuntu. It's just a few characters after "❌" in the list of emojis.

Reply:
That's why i said about casual text editor, not browsers. I'd prefer you commit such a change.

Reply:
No problem. Where is it in the code? I couldn't find it.

Reply:
`options.html` -> `id="add_domain"` -> `value=`

Reply:
Committed into this PR (I used "vim").